llvm/lib/Target/AMDGPU/AMDGPUInstructionSelector.cpp

```diff
@@ -97,9 +98,11 @@ bool AMDGPUInstructionSelector::isVCC(Register Reg,
 
 bool AMDGPUInstructionSelector::constrainCopyLikeIntrin(MachineInstr &MI,
                                                         unsigned NewOpc) const {
+  const bool NeedExec = NewOpc != AMDGPU::SI_READANYLANE;
```

Comment:
Why doesn't readanylane need exec?

Reply:
For the 1st version, it's SAlu and I thought it's uniform when defined. So...
Now, the intrinsic depends on exec.